ignite/pkg/xgit/xgit.go

```diff
@@ -13,7 +13,7 @@ var (
 	commitMsg  = "Initialized with Ignite CLI"
 	devXAuthor = &object.Signature{
 		Name:  "Developer Experience team at Tendermint",
```

Comment:
```suggestion
		Name:  "Developer Experience team at Ignite",
```
?

Reply:
Sure, I mean I thought maybe we didn't want to have many email addresses, but I guess it's not difficult to set them up.